WebPanel/pages/_app.tsx

```diff
@@ -1,138 +1,249 @@
-import { Dashboard as DashboardIcon, Home as HomeIcon, Laptop as LaptopIcon, Logout as LogoutIcon, Menu as MenuIcon, NetworkCheck as NetworkIcon, NotificationsNone as NotificationIcon, Settings as SettingsIcon } from '@mui/icons-material';
-import { AppBar, Box, Button, CssBaseline, Divider, Drawer, IconButton, Link, List, ListItemButton, ListItemIcon, ListItemText, Toolbar, Typography } from '@mui/material';
-import { SessionProvider, signIn, signOut, useSession } from 'next-auth/react';
-import { AppProps } from 'next/app';
-import Head from 'next/head';
-import { useRouter } from 'next/router';
-import { ReactElement, useEffect, useState } from 'react';
-import { LoadingScreen } from '../components';
+import {
+    Dashboard as DashboardIcon,
+    Home as HomeIcon,
+    Laptop as LaptopIcon,
+    Logout as LogoutIcon,
+    Menu as MenuIcon,
+    NetworkCheck as NetworkIcon,
+    NotificationsNone as NotificationIcon,
+    Settings as SettingsIcon,
+} from "@mui/icons-material";
+import {
+    AppBar,
+    Box,
+    Button,
+    CssBaseline,
+    Divider,
+    Drawer,
+    IconButton,
+    Link,
+    List,
+    ListItemButton,
+    ListItemIcon,
+    ListItemText,
+    Toolbar,
+    Typography,
```

Comment:
Formatters cannot enforce such code style, or can they?
If so I'd love to request a formatting configuration, otherwise, the formatting change is useless.